lib/rollbar/item/frame.rb

```diff
@@ -105,7 +105,8 @@ def pre_data(file_lines, lineno)
         to_line = lineno - 2
         from_line = [to_line - MAX_CONTEXT_LENGTH + 1, 0].max
 
-        file_lines[from_line, (to_line - from_line + 1)].select(&:present?)
+
+        file_lines[from_line, (to_line - from_line + 1)].select { |line| line && !line.empty? }
```

Comment:
So the two cases are `line == nil` or `line` a string. If `line` is nil then the first half of the conditional fails and we do not select it, otherwise `line` will have an `empty?` method, so `!line.empty?` will be equivalent to `present?`. Active support implements `present?` as `!blank?` and `blank?` as `responds_to(:empty?) ? !!empty? : !self` which is maybe a little bit safer because it works on any object not just something with an `empty?` method defined. But we can assume here that `line` is of a primitive type like string, array, hash that has an `empty?` method defined.